app/shop.py

```diff
@@ -0,0 +1,42 @@
+import datetime

```

Comment:
According to the checklist, you should use `import datetime` exactly as specified. Your import statement is correct.